src/amqp_multiplexer_test.ts

```diff
@@ -23,7 +23,8 @@ function createSocket() {
 }
 
 function sleep(ms: number) {
-  return new Promise((resolve) => setTimeout(resolve, ms));
+  const end = Date.now() + ms * 1_000;
+  while (Date.now() < end);
```

Comment:
I had a look at how this is used, and can see it is only used to "flush" or "tick" the event loop so that the new task is executed. Therefore, we can remove the `ms` argument and rename it to `flush` or similar. It should still return a promise so that is it clear that we need to await it at call site.

```
function flush() {
   return Promise.resolve();
}
```

Then replace the `await sleep(0)` with `await flush()`.